README.md

```diff
@@ -47,7 +47,7 @@ Then deploy an app. The example below creates a VM with a new volume:
           user: myname
           customizers:
           - $brooklyn:object:
-              type: brooklyn.location.blockstore.ec2.Ec2NewVolumeCustomizer
+              type: io.brooklyn.blockstore.brooklyn-blockstore:brooklyn.location.blockstore.NewVolumeCustomizer
```

Comment:
The changes here assume Karaf distribution, while the instructions above still document how to add dependencies to the classic distribution.
Ideally this should be added as a catalog item with `catalog.libraries` pointing to the blockstore jar. Alternatively could instruct the user to copy to `deploy` folder.
Do you think above instructions should be updated for karaf?